### PR TITLE
Fix confirmation screen

### PR DESCRIPTION
**Pull request by bgits**

### CODE REVIEW

File: app/components/ens/nameLookup.js [hunk 1]
```diff
@@ -251,7 +251,7 @@ class Register extends PureComponent {
              domainName={formattedDomainArray.slice(1).join('.')}
              domainPrice={domainPrice}
              preRegisteredCallback={() => { this.setState({ submitted: true }) }}
-             registeredCallbackFn={this.onRegistered} />}
+             registeredCallbackFn={console.log} />}
```
Comment (jeluard) [inline app/components/ens/nameLookup.js]: Is that the fix?

**Comment by jeluard** [comment]
You probably want to squash your commits

File: app/components/ens/nameLookup.js (hunk 1 above)
Comment (bgits) [reply app/components/ens/nameLookup.js]: Yes, I thought about removing the `registeredCallbackFn` but left it for both debugging and to give some options until there is some more clarity on the direction we move this DApp from here.

**Comment by bgits** [comment]
One more thing that came to mind reviewing this PR. This component can actually be a web3 UI component as `preRegisteredCallback` and `registeredCallbackFn` can support an optimistic UI for ENS, we just need to add a `registrationFailed` prop and that will allow the UI to update instantly when the user submits a TX but rollback and inform the user that the transaction actually failed.

**Pull request merged**